mobile/src/components/course/components/edit-course-category/edit-course-category.tsx

```diff
@@ -2,47 +2,31 @@ import React, { FC } from 'react';
 
 import { AppScreenName, DataStatus } from '~/common/enums/enums';
 import { CourseUpdateCategoryRequestDto } from '~/common/types/types';
-import {
-  BackButton,
-  Button,
-  Dropdown,
-  Spinner,
-  View,
-} from '~/components/common/common';
+import { BackButton, Spinner, View } from '~/components/common/common';
+import { EditCategoryDropdown } from '~/components/course/components/components';
 import {
   useAppDispatch,
-  useAppForm,
   useAppNavigate,
   useAppSelector,
   useEffect,
 } from '~/hooks/hooks';
-import { coursesActions } from '~/store/actions';
-import { courseUpdateCategory as courseUpdateCategoryValidationSchema } from '~/validation-schemas/validation-schemas';
+import { categoryActions, coursesActions } from '~/store/actions';
 
 import { styles } from './styles';
 
 const EditCourseCategory: FC = () => {
```

Comment:
```suggestion
const EditCourse: FC = () => {
```

Reply:
Please remove EditCourse component and reuse EditCourseCategory(with renaming it to EditCourse)